packages/core/database/src/metadata/index.ts

```diff
@@ -25,11 +25,11 @@ export {
 /**
  * Create Metadata from models configurations
  */
-export const createMetadata = (models: Model[] = []): Metadata => {
+export const createMetadata = (models: Model[], options: MetadataOptions): Metadata => {
```

Comment:
Do we have default values for the metadata options? Or should this always be defined manually?

Reply:
We made it required, because accidentally calling it without options in even one place could cause data loss. In a later refactoring I want to remove MetadataOptions entirely, because it should be possible to do something like instantiate an identifiers object in database with a fixed max length and export that to be used everywhere, but I couldn't come up with a clean pattern for it and ran out of time. But that cleanup can be done without breaking changes.